sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/cryptography/encrypt_parameters.hpp

```diff
@@ -24,8 +24,17 @@ namespace Azure {
    * @brief Parameters for encrypting plaintext.
    *
    */
-  struct EncryptParameters final
-  {
+  class EncryptParameters final {
```

Comment:
Essentially all non-result/option types should be structs.

For example, other types that should be class instead of struct:
`KeyVaultKey`
`CryptographyProvider` and its derived types

The rest seem reasonable (at least from what got generated in the latest APIView), it might be worth doing a quick review for all of key vault for such struct/class changes.